<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructElement</class>
 <widget class="QDialog" name="StructElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set kernel</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tama√±o&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tipo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rectangular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cruz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Elipse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diamente </string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
